components/tabs/test/tabs.visual-diff.html

```diff
@@ -48,6 +48,23 @@
 		</d2l-tabs>
 	</div>
 
+	<div class="visual-diff">
+		<d2l-tabs id="skeleton" skeleton>
+			<d2l-tab-panel text="All">Tab content for All</d2l-tab-panel>
+			<d2l-tab-panel text="Biology">Tab content for Biology</d2l-tab-panel>
+			<d2l-tab-panel text="Physics">Tab content for Physics</d2l-tab-panel>
+			<d2l-tab-panel text="Chemistry">Tab content for Chemistry</d2l-tab-panel>
+		</d2l-tabs>
+	</div>
+
+	<div class="visual-diff">
+		<d2l-tabs id="ellipsis">
```

Comment:
This is a new extra test I'm including here. A scenario we previously didn't have covered